hack/validate-boskos.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# This script ensures that the Boskos configuration checked into git is up-to-date
+# with the generator. If it is not, re-generate the configuration to update it.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+base_dir=.
+
+cd "${base_dir}/core-services/prow/02_config"
+./generate-boskos.py
+DIFF="$(git diff)"
```

Comment:
[Grr][1]:

```
hack/validate-boskos.sh: line 14: git: command not found
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/11752/rehearse-11752-pull-ci-openshift-release-master-boskos-config/1314310508639162368

Reply:
I've pushed 5cf5ba3f87 -> 204f9a738a to drop the `git` dependency.